mail/aerc/files/patch-makefile.diff

```diff
@@ -0,0 +1,11 @@
+--- Makefile.orig	2019-06-03 13:56:08.000000000 +0200
++++ Makefile	2019-06-04 12:16:08.000000000 +0200
+@@ -2,7 +2,7 @@
+ .SUFFIXES:
+ .SUFFIXES: .1 .5 .7 .1.scd .5.scd .7.scd
+ VPATH=doc
+-PREFIX?=/usr/local
++PREFIX?=/opt/local
```

Comment:
The MacPorts prefix is not guaranteed to be /opt/local, so this should not be hardcoded. See https://trac.macports.org/wiki/PortfileRecipes#hardcode_opt_local